src/controllers/trip.controller.js

```diff
@@ -58,7 +58,8 @@ class TripController {
     const offset = (page - 1) * limit;
 
     const results = await TripService.findByPropertyAndCountAll({ userId }, { offset, limit });
-
+    // const statuses = (results.rows).map(singleRes => singleRes.dataValues.request.status);
+    // console.log('^^^^^^^STATUSES LIST:', statuses);
```

Comment:
@higustave-ops Kindly remove these commented lines.

Reply:
@bdushimi i have implemented the feedbaxk